app/models/form_profile.rb

```diff
@@ -418,9 +420,23 @@ def clean!(value)
     end
   end
 
+  # def clean_hash!(hash)
+  #   # hash.deep_transform_keys! { |k| k.camelize(:lower) }
+  #   # hash.each { |k, v| hash[k] = clean!(v) }
+  #   # hash.delete_if { |_k, v| v.blank? }
+  # end
+  
   def clean_hash!(hash)
-    hash.deep_transform_keys! { |k| k.camelize(:lower) }
-    hash.each { |k, v| hash[k] = clean!(v) }
-    hash.delete_if { |_k, v| v.blank? }
+    hash.deep_transform_keys! do |key|
+      # convert key to string if it is a symbol
+      key = key.to_s if key.is_a?(Symbol)
+      key.camelize(:lower)
+    end
+
+    hash.each do |key, value|
+      hash[key] = clean!(value)
+    end
+  
+    hash.delete_if { |_key, value| value.blank? }
```

Comment:
Can you explain why you had to do this change here? If there this is a better solution, then I would say lets remove the commented section above.

Reply:
I added the functionality to convert Symbol to string because when I ran my test, I was running into an error stating `undefined method `camelize' for an instance of Symbol`
The keys were not being read as strings.

Reply:
This has been resolved. The error no longer shows and I reverted the function back to how it was originally and deleted the functionality to convert Symbol to string.